av1-rtp-spec.md

```diff
@@ -785,6 +799,28 @@ ns(n) {
 }
 </code></pre>
 
+The serialization proccess for this descriptor is specified as:
+
+<pre><code>
+write_ns(n,v) {
+  if (n == 1) return
+  w = 0
+  x = n
+  while (x != 0) {
+    x = x >> 1
+    w++
+  }
+  m = (1 << w) - n
+  if (v< m)
+    write(w - 1, val);
+  else
+    write(w, v + m) 

```

Comment:
same as function write_ns is opposite to ns,
function write is opposite to f.
may be stress it by renaming "write" to "write_f"

Reply:
done